scripts/check.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
+# Run flake8 against the Django codebase
+
 set -e
 set -x
 
-# Run flake8 against the Django codebase and output a known string so that
```

Comment:
You are totally right. In fact, the ``|| echo flake8...`` shoud be removed entirely (but that is out of the scope of this change)